### ISSUE TITLE
Outdated `dub build` commands when saving the file

### ISSUE THREAD

**Issue by aminya** [Pure-D/atom-ide-d#50]
It seems that `serve-d` runs dub build on save, but it uses some outdated sdl file. 

This happens with serve-d.beta-5. I haven't tested beta.6 due to #48 

I used `pnpm install --prod` in my sdl file before, but later I removed it. But it seems wherever I save my project, it stills runs that command. This happens across different sessions!

**Comment by WebFreak001** [comment]
serve-d loads the dub.sdl/dub.json into memory, it only updates it when reloading (which it does when you save the file and notify serve-d about it)

however it always needs to load the file again when restarting serve-d, so if the bug persists across sessions it sounds like something is still left on the harddrive.

I don't think anything about this has changed with the new serve-d beta compared to the previous one.

**Comment by aminya** [comment]
I think I found the issue. It is pnpm and not serve-d. [pnpm install](https://pnpm.io/cli/install#--prod--p) use `NODE_ENV` to decide if it use `--prod` or do a full installation. I noticed this issue because I had changed my command to `pnpm install`, but it still kept only installing `--prod` dependencies. Probably `NODE_ENV` is set to `production` by atom somewhere, which makes `pnpm install` be the same as `pnpm install --prod`.

As soon as I remove `pnpm install` from my command, the issue goes away.
